public/app/features/datasources/state/navModel.ts

```diff
@@ -16,7 +16,6 @@ export function buildNavModel(dataSource: DataSourceSettings, plugin: GenericDat
   const navModel: NavModelItem = {
     img: pluginMeta.info.logos.large,
     id: 'datasource-' + dataSource.uid,
-    subTitle: `Type: ${pluginMeta.name}`,
```

Comment:
Just for confirmation - are we sure we would like to remove the "Type: Prometheus" as a subtitle from the datasources header? (This is what tells the user what type of datasource they are looking at - besides the logo).

Reply:
While it's not a subtitle, it's still showing up as part of the page info. Is there a concern with having it as a page info vs subtitle?

This is mostly to match the new page header component. I agree that if "type" was the only piece of info it could live elsewhere.

Reply:
Sorry, I somehow missed that it is part of the info section now. No, I think it's fine if we have it in there 👍